files/en-us/web/css/flex-basis/index.html

```diff
@@ -14,7 +14,17 @@
 
 <div>{{EmbedInteractiveExample("pages/css/flex-basis.html")}}</div>
 
-<div class="note">
+<p>In this example the {{cssxref("flex-grow")}} and {{cssxref("flex-shrink")}} properties are both set to <code>1</code> on all three items, indicating that the flex item can grow and shrink from the initial <code>flex-basis</code>.</p>
+
+<p>The demo then changes the <code>flex-basis</code> on the first item. It will then grow and shrink from that flex-basis. This means that, for example, when the <code>flex-basis</code> of the first item is <code>200px</code>, it will start out at 200px but then shrink to fit the space available with the other items being at least <code>min-content</code> sized.</p>
+
+<figure style="padding-bottom: 2em;">
+  <img src="firefox-flex-basis.jpg" alt="The Firefox Flexbox Inspector showing the size of the item once it has shrunk">
```

Comment:
```suggestion
  <img src="firefox-flex-basis.jpg" alt="The Firefox Flexbox Inspector showing the size of the item once it has shrunk.">
```

Periods supply a nice pause for AT to help communicate the end of the alt description.